crates/biome_css_parser/src/syntax/at_rule/tailwind.rs

```diff
@@ -0,0 +1,240 @@
+use crate::lexer::CssLexContext;
+use crate::parser::CssParser;
+use crate::syntax::block::{
+    parse_declaration_block, parse_declaration_or_rule_list_block, parse_rule_block,
+};
+use crate::syntax::parse_error::{expected_identifier, expected_string};
+use crate::syntax::selector::parse_selector;
+use crate::syntax::{is_at_identifier, parse_identifier, parse_regular_identifier, parse_string};
+use biome_css_syntax::CssSyntaxKind::{self, *};
+use biome_css_syntax::T;
+use biome_parser::parse_lists::ParseNodeList;
+use biome_parser::parse_recovery::ParseRecoveryTokenSet;
+use biome_parser::parsed_syntax::ParsedSyntax;
+use biome_parser::parsed_syntax::ParsedSyntax::{Absent, Present};
+use biome_parser::prelude::*;
+
+// @theme { --color-primary: #3b82f6; }
+pub(crate) fn parse_theme_at_rule(p: &mut CssParser) -> ParsedSyntax {
+    if !p.at(T![theme]) {
+        return Absent;
+    }
+
+    let m = p.start();
+
+    p.bump(T![theme]);
+    if p.at(T![ident]) {
+        parse_regular_identifier(p).ok();
+    }
```

Comment:
Do we need to check `p.at(T![ident])`?

```suggestion
    parse_regular_identifier(p).ok();
```

Reply:
Oh, then yes this would make sense